src/lib.rs

```diff
@@ -1,23 +1,87 @@
-use std::error::Error;
 use clap::Parser;
+use futures::try_join;
+use reqwest::{Client, Response};
+use serde_json::{json, Value};
+use std::error::Error;
+use std::process;
 
 #[derive(Parser, Debug)]
 pub struct Args {
-    #[arg(short,long, help("Message to encrypt"))]
-    pub encrypt: Option<String>, 
-    #[arg(short,long, help("Link to decrypt message from"))]
+    #[arg(short, long, help("Message to encrypt"))]
+    pub encrypt: Option<String>,
+
+    #[arg(
+        short,
+        long,
+        help("Time of expiry after encrypting a message"),
+        default_value_t = 50
+    )]
+    pub time_to_live: u8,
+
+    #[arg(short, long, help("Room id to decrypt message from"))]
```

Comment:
Just fyi, if you want to have encrypt and decrypt be mutually exclusive, [this](https://docs.rs/clap/4.0.18/clap/_derive/_tutorial/index.html#subcommands) should be the way to do it I think